admin/class-yoast-dashboard-widget.php

```diff
@@ -38,6 +38,7 @@ public function __construct( WPSEO_Statistics $statistics = null ) {
 	 * Adds dashboard widget to WordPress
 	 */
 	public function add_dashboard_widget() {
+		add_filter( 'postbox_classes_dashboard_wpseo-dashboard-overview', array( $this, 'wpseo_dashboard_overview_class' ) );
```

Comment:
Sure it can be removed, then the dashboard widget should be targeted separately since it would not inherit styles from the `.yoast` selector. What about the similar approach in `class-metabox.php`? It uses a filter to add the CSS class, is that OK?